pkg/apis/monitoring/v1beta1/validation.go

```diff
@@ -183,12 +183,12 @@ func (yr YearRange) Parse() (*ParsedRange, error) {
 
 	start, err := strconv.Atoi(startStr)
 	if err != nil {
-		fmt.Errorf("start year cannot be %s parsed: %w", startStr, err)
+		return nil, fmt.Errorf("start year cannot be %s parsed: %w", startStr, err)
```

Comment:
I'm not sure why those were ignored, maybe I should not have fixed them in this PR